python/ray/autoscaler/v2/utils.py

```diff
@@ -349,58 +353,67 @@ def format(cls, data: ClusterStatus, verbose: bool = False) -> str:
         return formatted_output.strip()
 
     @staticmethod
-    def _node_usage_report(data: ClusterStatus, verbose: bool) -> str:
-        usage_by_node = {}
-        node_type_mapping = {}
-        idle_time_map = {}
+    def _node_usage_report(
+        active_nodes: List[NodeInfo], idle_nodes: List[NodeInfo]
+    ) -> str:
+        """[Example]:
+        Node: raycluster-autoscaler-small-group-worker-n8hrw (small-group)
+         Id: cc22041297e5fc153b5357e41f184c8000869e8de97252cc0291fd17
+         Usage:
+          1.0/1.0 CPU
+          0B/953.67MiB memory
+          0B/251.76MiB object_store_memory
+         Activity:
+          Resource: CPU currently in use.
+          Busy workers on node.
+        """
+        node_id_to_usage = {}
+        node_id_to_type = {}
+        node_id_to_idle_time = {}
+        node_id_to_instance_id = {}
 
         # Populate mappings for usage, node types, and idle times
-        for node in chain(data.active_nodes, data.idle_nodes):
-            usage_by_node[node.node_id] = {
+        for node in chain(active_nodes, idle_nodes):
+            node_id_to_usage[node.node_id] = {
                 u.resource_name: (u.used, u.total) for u in node.resource_usage.usage
             }
-            node_type_mapping[node.node_id] = node.ray_node_type_name
-            idle_time_map[node.node_id] = node.resource_usage.idle_time_ms
-
+            node_id_to_type[node.node_id] = node.ray_node_type_name
+            node_id_to_idle_time[node.node_id] = node.resource_usage.idle_time_ms
+            node_id_to_instance_id[node.node_id] = node.instance_id
         # Create a dictionary for node activities
-        node_activities = {
-            node.node_id: node.node_activity for node in data.active_nodes
-        }
+        node_activities = {node.node_id: node.node_activity for node in active_nodes}
 
         node_usage_report_lines = []
-        if verbose:
-            if usage_by_node:
-                for node_id, usage in usage_by_node.items():
-                    node_usage_report_lines.append("")  # Add a blank line between nodes
-
-                    # Node type line
-                    node_type_line = f"Node: {node_id}"
-                    if node_id in node_type_mapping:
-                        node_type = node_type_mapping[node_id]
-                        node_type_line += f" ({node_type})"
-                    node_usage_report_lines.append(node_type_line)
-
-                    # Idle time for the node
-                    if idle_time_map.get(node_id, 0) > 0:
-                        node_usage_report_lines.append(
-                            f" Idle: {idle_time_map[node_id]} ms"
-                        )
+        for node_id, usage in node_id_to_usage.items():
+            node_usage_report_lines.append("")  # Add a blank line between nodes
```

Comment:
this adds a blank line before printing the first node's information -- does that make the formatting weird?

Reply:
> does that make the formatting weird?

It makes it easier to read in my opinion. It's the line between "Busy workers on node" and "Node: ..." in the screenshot I attached in the PR description.